### PR TITLE
Update knowledge-submissions-past-wikipedia.md

### LINKED ISSUE #59: need recommendations for trusted data sources beyond Wikipedia for Knowledge submissions
For background on this issue, please review this document on [InstructLab knowledge submissions beyond Wikipedia](https://github.com/instructlab/dev-docs/blob/main/docs/knowledge-submissions-past-wikipedia.md). 

There are any number of other domains we may wish to add as approved content providers. Please use this issue to make recommendations for review by legal experts.

### PR DESCRIPTION
**Pull request by lhawthorn**
Added new approved Knowledge submission data sources, updated status for several.

Updated documentation with new process to take in requested knowledge sources to be open a PR against this devdoc.

Related to issue #59 which should be closed once this PR is reviewed and merged.

Closes #59

### CODE REVIEW

**Comment by jjasghar** [comment]
I helped out yoru linting issues, the spell check doesn't like a bunch of the words...i guess we'll need to add them to the spell checker now:
```
Cynefin
Knowlege
OpenStax
PLOS
Standup
butvis
io
```

File: docs/knowledge-submissions-past-wikipedia.md [hunk 1]
```diff
@@ -24,27 +24,41 @@ Status:
 - `denied`: Denied by the legal team, and posted on the [avoided list][avoided].
 - `submitted`: Sent to the legal team for review
 - `proposed`: The community would like to propose this as a possible place to take knowledge submissions from.
+- `reviewed - manually verify`: Legal team has reviewed this domain and while much of its source material meets our open licensing criteria, not all of it does. Each submission from this source must be manually verified to actually be under an appropriate content license or e.g. definitively in the public domain.
+
+For the purposes of Knowlege submissions to the InstructLab project, data sourced from items in the `approved` category require no further vetting from the Triage and/or other Maintainer teams. Items in the `reviewed - manually verify` category will require vetting before the submission can be accepted.
```
Comment (markstur) [inline docs/knowledge-submissions-past-wikipedia.md:29]: s/Knowlege/Knowledge/

File: docs/knowledge-submissions-past-wikipedia.md [hunk 2]
```diff
@@ -24,27 +24,41 @@ Status:
...
+| Internet Archive: <https://archive.org/> | reviewed - manually verify | Pre-1927 works; public domain under US copyright law |
+| PLOS family of open access journals: <https://plos.org/publish/> | reviewed - manually verify  | |
+| Open Practice Library: <https://openpracticelibrary.com/>  | reviewed - manually verify  | |
+| Cynefin.io wiki: <https://cynefin.io/wiki/Main_Page>  | reviewed - manually verify  | |
+| The Open Education Project: <https://research.redhat.com/blog/research_project/foundations-in-open-source-education/>  | reviewed - manually verify  | |
+
+### Process steps
+
+1. Collect suggested places from the community by requesting they submit a pull request against this dev doc.
+1. Work with our legal team to adjudicate. [@lhawthorn](https://github.com/lhawthorn) is currently the owner of this step, butvis happy to educate & empower other folks to do this work.
```
Comment (markstur) [inline docs/knowledge-submissions-past-wikipedia.md:57]: typo s/butvis/but is/

**Comment by lhawthorn** [comment]
LH is unaware of the existence of a Mergify dashboard and will get educated when maintainer team is online.

**Comment by lhawthorn** [comment]
@mergifyio rebase

**Review by markstur — APPROVED** [review]
LGTM

File: docs/knowledge-submissions-past-wikipedia.md [hunk 3]
```diff
@@ -24,27 +24,41 @@ Status:
 - `denied`: Denied by the legal team, and posted on the [avoided list][avoided].
 - `submitted`: Sent to the legal team for review
 - `proposed`: The community would like to propose this as a possible place to take knowledge submissions from.
+- `reviewed - manually verify`: Legal team has reviewed this domain and while much of its source material meets our open licensing criteria, not all of it does. Each submission from this source must be manually verified to actually be under an appropriate content license or e.g. definitively in the public domain.
```
Comment (bjhargrave) [inline docs/knowledge-submissions-past-wikipedia.md:27]:
```suggestion
- `reviewed - manually verify`: Legal team has reviewed this domain and while much of its source material meets our open licensing criteria, not all of it does. Each submission from this source must be manually verified to actually be under an appropriate content license or definitively in the public domain.
```

File: docs/knowledge-submissions-past-wikipedia.md [hunk 4]
```diff
@@ -24,27 +24,41 @@ Status:
...
+| Cynefin.io wiki: <https://cynefin.io/wiki/Main_Page>  | reviewed - manually verify  | |
+| The Open Education Project: <https://research.redhat.com/blog/research_project/foundations-in-open-source-education/>  | reviewed - manually verify  | |
+
+### Process steps
+
+1. Collect suggested places from the community by requesting they submit a pull request against this dev doc.
+1. Work with our legal team to adjudicate. [@lhawthorn](https://github.com/lhawthorn) is currently the owner of this step, but is happy to educate & empower other folks to do this work.
+1. Inform the triage team and triagers of the new locations we can or can not accept. This is currently done via an announcement in the [daily Triager Standup](https://github.com/instructlab/community/blob/main/Collaboration.md#triager-standup) and via a pull request to update the Knowledge Guide in one of the two locations listed below.
+
+- Approved sources: <https://github.com/instructlab/taxonomy/blob/main/docs/KNOWLEDGE_GUIDE.md#accepted-knowledge>
```
Comment (bjhargrave) [inline docs/knowledge-submissions-past-wikipedia.md:60]:
```suggestion
- [Approved sources][approved]
```

File: docs/knowledge-submissions-past-wikipedia.md [hunk 5]
```diff
@@ -24,27 +24,41 @@ Status:
...
+- Rejected sources: <https://github.com/instructlab/taxonomy/blob/main/docs/KNOWLEDGE_GUIDE.md#avoid-these-topics>
```
Comment (bjhargrave) [inline docs/knowledge-submissions-past-wikipedia.md:61]:
```suggestion
- [Rejected sources][avoided]
```

**Review by bjhargrave — CHANGES_REQUESTED** [review]

**Review by nathan-weinberg — APPROVED** [review]
Pending @bjhargrave's comments this LGTM

File: docs/knowledge-submissions-past-wikipedia.md (hunk 3 above)
Comment (hickeyma) [reply docs/knowledge-submissions-past-wikipedia.md:27]: Updated

**Comment by bjhargrave** [comment]
@hickeyma Your fix commit is not pushed to this PR. It is just in your fork.

**Review by hickeyma — APPROVED** [review]
LGTM thanks @lhawthorn  for the update of the updates. 

@bjhargrave Added your suggestions. Do you mind doing a quick review?

**Comment by hickeyma** [comment]
Shoot, I see that @bjhargrave. Let me try and fix that.

**Comment by hickeyma** [comment]
Thanks @bjhargrave for the help with the initial commit. Ready for review now.

**Review by bjhargrave — APPROVED** [review]

**Pull request merged**